tests/integration/test_validate_config.py

```diff
@@ -241,6 +309,6 @@ def test_schema_validation(tmpdir, raw_package_config, valid, expected_output):
             output = PackitAPI.validate_package_config(Path("test_dir"))
             assert expected_output in output
         else:
-            with pytest.raises(PackitConfigException) as exc:
+            with pytest.raises(Exception) as exc_info:
```

Comment:
Why the change to generic `Exception`?

Reply:
I don't remember and doesn't seem to be needed. I will revert it 😅